src/test/java/org/commcare/formplayer/tests/MediaCaptureTest.kt

```diff
@@ -93,7 +102,12 @@ class MediaCaptureTest {
         var expectedFilePath = getExpectedMediaPath(formResponse.session_id, responseBean)
         val originalSavedFile = expectedFilePath.toFile()
         assertTrue("Could not find saved file on the filesystem", originalSavedFile.exists())
-
+        // check that metadata was created and values match expected
+        val fileName = expectedFilePath.fileName.toString()
+        val metadataId = fileName.substring(0, fileName.indexOf("."))
+        val originalMetadata = mediaMetaDataService.findById(metadataId)
```

Comment:
can we assert for all properties of `originalMetadata` atleast once during this test so that we know all properties are getting saved correctly. It would be ideal to create a new `private` function and do it there for the sake of readability.